packages/node/test/detection-fixtures/node/index.ts

```diff
@@ -0,0 +1,3 @@
+export default function (req, res) {
```

Comment:
Should we add types here to ensure its detected with real TS code?

Reply:
Also another test with `req` but not `res`?

Reply:
It makes no difference to the feature detector (which has its [own tests](https://github.com/vercel/edge-runtime/blob/main/packages/feature-detector/test/has-edge-signature.test.ts)), but it can only be beneficial to add more. Doing it.